frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -11,10 +11,8 @@ modal-base-template(ref='modal' :fullscreen='true' class='has-background' v-if='
         novalidate='true'
       )
         i18n.has-text-bold.c-title(tag='h3') Who did you send money to?
-        payments-list(
-          :titles='tableTitles'
-          :payments='paymentsDistribution'
-          paymentsType='edit'
+        record-payments-list(
+          :paymentsList='ephemeral.payments'
```

Comment:
Scouting rule: Add css class`.textarea` to `textarea` on line 39. The styles are missing there.